### ISSUE TITLE
Check for multiple timezones not working properly

### ISSUE THREAD

**Issue by Platzii** [laidig/gtfs-validator#3]
Hi

Seems like the check for multiple timezones does not work properly. Im getting this output:
```
Exception in thread "main" java.lang.IllegalArgumentException: File contains two time zones, which is not allowed by the GTFS spec
    at com.conveyal.gtfs.service.CalendarDateVerificationService.<init>(CalendarDateVerificationService.java:82)
    at com.conveyal.gtfs.validator.ValidatorMain.main(ValidatorMain.java:70)
```

The agency.txt file contains multiple agencies but they are all in the same timezone as seen here:
<img width="664" alt="screen shot 2017-01-20 at 11 01 47" src="https://cloud.githubusercontent.com/assets/7767841/22145023/f0ed0b68-deff-11e6-9efc-f7fc3e585423.png">
Used feed: http://gtfs.ovapi.nl/new/archive/NL-20170119.gtfs.zip 

Kind regards
Simon

**Comment by laidig** [comment]
It's this from the same feed?

**Comment by Platzii** [comment]
The error occurred when using this feed: http://gtfs.ovapi.nl/new/archive/NL-20170119.gtfs.zip

**Comment by laidig** [comment]
Got it. These European rail feeds are sure interesting.

**Comment by Platzii** [comment]
Seems like the problem is fixed by commit ed0a8acd254f76c564fee17eacf866d010d2fd3c
Nice job!
